src/context/realTimeDataTrackerContext.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { useState, useEffect, createContext } from "react";
+
+export const RealTimeDataTrackerContext = createContext(null);
```

Comment:
because it will share like count and other stats important to have to all components that will be updated from different places on change which makes it basically real time traceker.